lambdas/service/app.py

```diff
@@ -319,19 +319,29 @@ def validate_params(query_params: Mapping[str, str],
 @app.route('/integrations', methods=['GET'], cors=True)
 def get_integrations():
     query_params = app.current_request.query_params or {}
-    validate_params(query_params, entity_type=str, integration_type=str)
+    validate_params(query_params, entity_type=str, integration_type=str, entity_ids=str)
+    entity_ids = query_params.get('entity_ids', '')
+    entity_ids = None if entity_ids == '' else set(entity_ids.split(','))
     try:
         entity_type = query_params['entity_type']
         integration_type = query_params['integration_type']
     except KeyError:
         raise BadRequestError('Parameters entity_type and integration_type must be given')
-    body = _fetch_integrations(entity_type, integration_type)
+    body = _fetch_integrations(entity_type, integration_type, entity_ids)
     return Response(status_code=200,
                     headers={"content-type": "application/json"},
                     body=json.dumps(body))
 
 
-def _fetch_integrations(entity_type, integration_type):
+def _fetch_integrations(entity_type: str, integration_type: str, entity_ids: Set[str]) -> List[Dict[str, Any]]:
```

Comment:
https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#16type-hints

Reply:
IOW

```suggestion
def _fetch_integrations(entity_type: str, integration_type: str, entity_ids: Set[str]) -> Sequence[JSON]:
```

Reply:
Fixing with `Sequence[JSON]` however I noticed in types.py there is a `JSONs = Sequence[JSON]` that doesn't appear to be used anywhere in the codebase yet. Should we start using `JSONs` instead?

Reply:
Yes. Good catch.